connectors/spark/src/main/scala/io/unitycatalog/spark/auth/CredPropsUtil.java

```diff
@@ -153,16 +168,58 @@ private static Map<String, String> s3PathTempCredProps(
         .build();
   }
 
-  private static Map<String, String> gsProps(TemporaryCredentials tempCreds) {
+  private static Map<String, String> gsFixedCredProps(TemporaryCredentials tempCreds) {
     GcpOauthToken gcpOauthToken = tempCreds.getGcpOauthToken();
-    return ImmutableMap.<String, String>builder()
-        .put(GcsVendedTokenProvider.ACCESS_TOKEN_KEY, gcpOauthToken.getOauthToken())
-        .put(GcsVendedTokenProvider.ACCESS_TOKEN_EXPIRATION_KEY,
+    return new GcsPropsBuilder()
+        .set(GcsVendedTokenProvider.ACCESS_TOKEN_KEY, gcpOauthToken.getOauthToken())
+        .set(GcsVendedTokenProvider.ACCESS_TOKEN_EXPIRATION_KEY,
```

Comment:
Do we still need the `expiration key`  if it's a `fixedCredProps` ?   I will assume it's not.

Reply:
`fs.gs.auth.access.token.expiration` this key , is it an official key from google, or our customized key ? If it's later, then I think we can remove it now.